server/database/get-user-data.js

```diff
@@ -0,0 +1,47 @@
+const dbClient = require('./db_connection.js');
+const { formatUserGoals, formatUserRatings, } = require('../helpers/format-user-data');
+
+const getGoalsQuery = `
+  SELECT goals.user_id, goals.goal_id, goals.title, goals.icon
+  FROM goals
+  WHERE user_id=$1 AND goals.deleted=false
+`;
+
+const getRatingsQuery = `
+  SELECT
+    ratings.rating_id,
+    ratings.rating,
+    ratings.comment,
+    ratings.date_created
+  FROM ratings
+  WHERE user_id=$1 AND goal_id=$2
```

Comment:
I expect that if `goal_id`=`$2`, it is impossible that `user_id` != `$1`
ie `user_id=$1` is unnecessary